index.html

```diff
@@ -181,6 +189,9 @@ <h2 class="ma0 codystar f2 f1-ns dark-gray tc pt3"> Gift Registry </h2>
         <p class="m0 f6 tr pr4 white">If you have a questions <a href="mailto:mdc+wedding@adecoy.com" class='white'>drop us an email</a>.</p>
       </section>
 
+      <script data-cfasync="false" type="text/javascript"
+  src="https://cdn.rawgit.com/dwyl/html-form-send-email-via-google-script-without-server/master/form-submission-handler.js">
```

Comment:
Now would be lovely if you had time, this is what I was hoping we could do this morning!